geometry_msgs/Pose commanded
geometry_msgs/Pose measured
geometry_msgs/Pose error 